i18n/README-ZH.md

```diff
@@ -21,73 +20,78 @@
 
 ## 关于 1.0.0
 
-⚠**警告**⚠ : 当前为 Flame `1.0.0` 的候选版本. 尚未预备好投入生产(正式版), 并且此处的代码尚未根据代码进行完整更新. 此处全部的内容都可能在正式版发布前发生变更.
+我们的目标是尽快发布 v1 版本。
+随着开发工作的推进，我们将按照周期发布预览版本，并且目前来看我们的进度十分可观（但仍需要一些后续改动）。
 
-通过使用该版本, 可以抢先预览Flame的新版本情况, 你可以为团队提供新功能、新结构的反馈.
+我们希望你可以使用新版本的 Flame 进行体验，并且针对新的结构和功能给予我们团队反馈。
 
-> 本篇为社区版翻译, 目前针对版本: 0.29.0
+目前在 pub 上已发布的 v1 版本为
+<a title="Pub" href="https://pub.flutter-io.cn/packages/flame" ><img src="https://img.shields.io/pub/v/flame.svg?style=popout&include_prereleases" /></a>。
+最新的稳定版本为
+<a title="Pub" href="https://pub.flutter-io.cn/packages/flame" ><img src="https://img.shields.io/pub/v/flame.svg?style=popout" /></a>。
+请根据你的需要选择合适的版本。
 
 ---
 
 ## 文档
-请注意, 最新的代码不在最后的release版本中, 而是在`master`分支.
 
-你可以在此处找到各版本的文档:
+请注意，主分支的文档要比已发布的版本更新。
+
+你可以在此处找到各版本的文档：
 - 最新稳定版: 
-    - [Flame引擎官网](https://flame-engine.org/)
-    - [Github docs](https://github.com/flame-engine/flame/tree/master-v0.x/doc)
-    
-- v1.0.0 版本:
-    - [GitHub docs](https://github.com/flame-engine/flame/tree/1.0.0-rc6/doc)
-    
-- 中文社区版文档(非官方, 针对中文环境):
-    - [Flame中文网](https://www.flame-cn.com/)
-    - [Flame中文文档](https://docs.flame-cn.com/)
+    - [Flame 引擎官网](https://flame-engine.org/)
+    - [GitHub 文档](https://github.com/flame-engine/flame/tree/master-v0.x/doc)
 
-你可以在这里看到完整的[开发文档](doc/README.md)
+- v1.0.0 版本：
+    - [GitHub docs](https://github.com/flame-engine/flame/tree/1.0.0-releasecandidate.11/doc)
 
-你也可以在[这里](doc/examples)看到许多不同功能的示例(文档/demo), [点击此处](./example)来查看一个入门demo.
+- 中文社区版文档（非官方，针对中文环境）：
+    - [Flame 中文网](https://www.flame-cn.com/)
+    - [Flame 中文文档](https://docs.flame-cn.com/)
 
-Flame的官方网站(其中包含文档), 请[点击此处](https://flame-engine.org/)
+你可以在这里看到完整的 [开发文档](doc/README.md)。
 
-## 问题互助
+你可以在浏览器中尝试 [我们提供的一些示例](https://flame-engine.github.io/flame/)。
+你也可以在 [这里](https://github.com/flame-engine/flame/tree/main/examples) 查看示例的源码。
 
-我们在Fireslime的Discord上有一个互助频道, [点击加入](https://discord.gg/pxrBmy4).
+Flame 的官方网站（包含文档），请 [访问此处](https://flame-engine.org/)。
 
-我们也有[FAQ](FAQ.md), 所以提问前请先在这里看看你有没有需要的答案.
+## 问题互助
 
-国内用户欢迎加入[Flame QQ交流群](https://jq.qq.com/?_wv=1027&k=5ETLFm3)(非官方, 针对国内用户的讨论群)
+我们在 Discord 上有一个互助频道，[点击加入](https://discord.gg/pxrBmy4)。
+
+我们也有 [FAQ](FAQ.md)，所以在提问前，请先搜索是否有你需要的答案。
 
 ## 项目目标
 
-这个项目的目的是为使用Flutter进行开发的每个游戏都会遇到的常见问题提供一套完整的解决方案.
+该项目的目的是为使用 Flutter 开发的游戏会遇到的常见问题提供一套完整的解决方案。
 
-目前它提供了:
-- 游戏循环(game loop)
+目前项目提供了以下内容：
+- 游戏循环 (game loop)
 - 组件/对象系统
-- 内置物理引擎(box2d)
+- 内置物理引擎（Forge2D，通过 [flame_forge2d](https://github.com/flame-engine/flame_Forge2D) 提供）
 - 音频支持
 - 特效与粒子效果
 - 手势和输入支持
-- 图片, 精灵(sprite)以及精灵组
-- 基础Rive支持
-- 和一些简化开发的实用工具类
+- 图片、精灵图 (sprite) 以及精灵图组
+- 基础 Rive 支持
+- 一些简化开发的实用工具类
 
-你可以按需使用它们, 因为它们在某种程度上是相互独立的.
+你可以按需使用它们，某种程度上来说它们是相互独立的。
 
 ## 支持我们
 
-顺手点一颗Star, 就可以提供你的帮助!
+顺手点一颗 Star，就是你对我们的帮助！
 
-你也可以成为赞助者, 通过Patreon来支持我们:
+你也可以成为赞助者，通过 Patreon 来支持我们：
 
 [![Patreon](https://c5.patreon.com/external/logo/become_a_patron_button.png)](https://www.patreon.com/fireslime)
 
-或者通过买杯咖啡捐赠我们:
+或者通过 But Me A Coffet 捐赠我们：
```

Comment:
Maybe it's `Buy Me a Coffe`, not `But Me A Coffet`.

Reply:
It's been updated.